app/model/ClientConfig.scala

```diff
@@ -0,0 +1,15 @@
+package model
+
+import play.api.libs.functional.syntax._
+import play.api.libs.json.{JsValue, Json, JsPath, Format}
+
+case class ClientConfig(capiUrl: String, capiKey: String, tagTypes: List[String])
+
+object ClientConfig {
+
+  implicit val clientConfigFormat: Format[ClientConfig] = (
```

Comment:
The object? or the implicit val? 

Trying to find the cleanest way to JSON encode this information, which is going to expand as we pass more info to client on load.